### PR TITLE
en, zh: update doc about deploying initializer

### PR DESCRIPTION
**Pull request by KanShiori**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
file: [initialize-a-cluster.md](https://github.com/pingcap/docs-tidb-operator/blob/master/zh/initialize-a-cluster.md)
prompt the user to modify some required fields.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by csuzhangxc — APPROVED** [review]
LGTM

File: en/initialize-a-cluster.md [hunk 1]
```diff
@@ -17,6 +17,19 @@ This document describes how to initialize a TiDB cluster in Kubernetes (K8s), sp
 
 Refer to [TidbInitializer example](https://github.com/pingcap/tidb-operator/blob/master/manifests/initializer/tidb-initializer.yaml), [API documentation](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.md), and the following steps to complete TidbInitializer Custom Resource (CR), and save it to the `${cluster_name}/tidb-initializer.yaml` file. Please switch the TidbInitializer example and API documentation to the currently used version of TiDB Operator.
 
+Modify `spec.cluster.namespace` field and `spec.cluster.name` field in the file:
```
Comment (qiancai) [inline en/initialize-a-cluster.md:20]:
```suggestion
### Set the cluster namespace and name

In the `${cluster_name}/tidb-initializer.yaml` file, modify the `spec.cluster.namespace` and `spec.cluster.name` fields:
```

**Review by qiancai — COMMENTED** [review]
@KanShiori PTAL. Thanks!

File: zh/initialize-a-cluster.md [hunk 2]
```diff
@@ -17,6 +17,21 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/initialize-a-cluster/']
...
+
+```yaml
+# ...
+spec:
+  # ...
+  cluster:
+    namespace: ${cluster_namespace}
+    name: ${cluster_name}
+```
+
```
Comment (DanielZhangQD) [inline zh/initialize-a-cluster.md:34]: 不用配置 namespace，下面最终创建的 kubetl apply -n $ns 就可以，把 tidb-operator repo 示例中的 namespace 去掉。

Comment (KanShiori) [reply zh/initialize-a-cluster.md:34]: 这个是配置指向 TiDBCluster，不是资源的 namespace。示例中是 demo

Comment (DanielZhangQD) [reply zh/initialize-a-cluster.md:34]: 奥，看错了

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**